src/properties/text_property.rs

```diff
@@ -157,6 +157,7 @@ pub enum TextHistoryType {
 
 /// FText history
 #[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", serde_with::skip_serializing_none)]
```

Comment:
This avoids the need to annotate every `Option` struct field across all of the enum variants:
```rs
#[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
```